Turtlelord26 GravTech Updates!

Turtlelord26 opened the pull request:
Addressing your concerns about Conical Field ranges. May have scope crept a lot. Send help. Also look at the things!

Field shape ranges divided into 3 categories, now owned by Projector items instead of level. Save DCs moved there too.
First item entries for Cores and Peripherals. They give "Resource" instead of gravitons because **[REDACTED]** will also use them to generate **[REDACTED]**!

Turtlelord26 committed:
Rework Range and Save DC of Grav Techniques

Removed Standard Range and DCs by level. Both range and DC are now set by the Gravity Projector, a weapon equivalent for this class, and may be modified by cores and peripherals, which are Technician-specific equipment.

Added channeling rules and [coughs] concentration checks [coughs] Will saves to maintain channels under fire.

Sector and Cone field shape angles cut in half as part of a larger directed-AoE nerf.

Activation time increased one 1 to 2 actions across the board, since GravTechs don't have to aim.

Turtlelord26 committed:
GravTech Prose Updates

Redone wording to enable structural changes in nearby commits.

Added a couple of class feats for stabbity stab stab.

Turtlelord26 committed:
Create 04_07_Technician_Equipment

I don't know why or how I came up with the idea but unless someone stops me, every. single. core. is going to be riffing off of Intel. And I will be cackling the whole time.

Anyway, here's some equipment for Technicians. With stats and costs and everything. It's very shiny.

Turtlelord26 committed:
Typography

Turtlelord26 committed:
Added another peripheral

Turtlelord26 committed:
Update 04_07_Technician_Equipment

Added Reflex Modifier and Movement Speed Penalty stats to projectors.
Don't put your lv 1 gravtech on point when room clearing is all I'm saying. Also projectors, being full of weird paramassive materials, are heavy.

Turtlelord26 committed:
Added Field Shape Types

Now each shape is explicitly either Beam, Conical, or Radial, for divided ranges set by projectors.

Turtlelord26 committed:
Update to grav blades, use INT instead of STR

You don't have to be strong to stab with gravity, but smarts help with knowing where to put it.

bleehu commented on CharacterCreation/02_Classes.txt:
```diff
@@ -761,22 +761,28 @@ Passives:
-materials the Technician needs. This swarm cannot survive outside the Core and 
-individual nanites will slag themselves if removed. The Gravity Generator attached to 
-the Core for power and is approximately the size of a large assault rifle, requiring 
+	At level 1 your character starts with an M1-01 Kepler Lake Microassembly 
```
great name!

bleehu commented on CharacterCreation/02_Classes.txt:
```diff
@@ -761,22 +761,28 @@ Passives:
+its core and relies on the core's central processor to think; individual nanites will 
+slag themselves if removed from the core's environment. Gravity projectors attach to the 
+Core for power and are approximately the size of a large assault rifle, requiring 
 two hands to wield.
```
great descriptions! Reminds me of the crononaut from red alert 2...

bleehu commented on CharacterCreation/02_Classes.txt:
```diff
@@ -796,41 +802,60 @@ Skill Proficiencies:
+projectors. When projecting a technique through multiple generators, each generator 
+must be using the same field shape, and if applicable, must be directed toward the 
+same point. Generators must be separated by at least 5 meters to avoid intereference - 
+any closer and they won't activate.
```
fascinating! This is going to need some testing. Good area control design.

bleehu commented on CharacterCreation/02_Classes.txt:
```diff
@@ -796,41 +802,60 @@ Skill Proficiencies:
+Gravity Dagger:
+	You have miniaturized gravity blade technology into a module that can be held 
+in one hand. This is a separate device from the gravity projector and cannot be wielded 
+simultaneously with one. Gravity daggers can be dual-wielded. Increase the damage of 
```
with or without the dual-wield feat?

bleehu reviewed (approved):
Subject to further review of balance of abilities, but I think this is nearing test-ability. is there a note somewhere about how gravity fields interact with cover?

Turtlelord26 committed:
Clarity for grav dagger

Remove interpretation that they can be dual-wielded without the feat.

Turtlelord26 committed:
Add note on cover.

Your upturned jeep does not block gravity very well, but you are still a bit harder to see.

Turtlelord26 commented:
There is now. It only affects techniques that have to be aimed. You won't find many gravity-blocking materials in the wild.
